wallet/udb/txmined.go

```diff
@@ -2855,6 +2866,13 @@ func (s *Store) unspentOutputsForAmount(ns, addrmgrNs walletdb.ReadBucket, neede
 				syncHeight) {
 				return nil
 			}
+
+			if fetchRawCreditHasExpiry(cVal) {
```

Comment:
This check must be applied to all outputs, not just ticket purchase change: https://github.com/decred/dcrd/blob/fc102faefa56e4945c4cadbe029e0ae87470cc9b/blockchain/validate.go#L1708